src/components/cms/SkillFeedbackPage/EditFeedbackDialog.js

```diff
@@ -6,15 +6,35 @@ import DialogTitle from '@material-ui/core/DialogTitle';
 import OutlinedTextField from '../../shared/OutlinedTextField';
 import FormControl from '@material-ui/core/FormControl';
 import Button from '../../shared/Button';
+import Rating from '../../shared/Rating';
+import isMobileView from '../../../utils/isMobileView';
 
 const EditFeedback = props => {
-  const { handleConfirm, handleClose, feedback, handleEditFeedback } = props;
+  const {
+    handleConfirm,
+    handleClose,
+    feedback,
+    handleEditFeedback,
+    handleEditRating,
+    rating,
+  } = props;
 
   const [loader, updateLoader] = useState(false);
   const [newFeedback, updateNewFeedback] = useState(feedback);
+  const [newRating, updateRating] = useState(rating);
+  const mobileView = isMobileView();
+  const RATING_TO_HINT_MAP = new Map([
```

Comment:
Put it under constant folder. And reuse it.

Use syntax similar to usage across repo - 
```
const RATING_TO_HINT_MAP = {
0: 'The skill has not been rated by you',
1: 'Hated it',
...
};
```